README.md

```diff
@@ -233,13 +233,7 @@ import someSvg from './some.inline.svg';
 
 **Should Node.js be responsible for compressing assets with gzip?**
 
-If possible, you should do compression in an upstream server like **nginx**. Doing the compression in **Node.js** might hold the event loop which it not desirable for performance reasons.
-
-To disable `gzip`, you may start the production server with:
-
-```sh
-npm start -- --no-gzip
-```
+We use [compression-webpack-plugin](https://github.com/webpack-contrib/compression-webpack-plugin) to compress the assets when the project is built. When starting a production server, you can choose to serve the compressed assets (by default) or the regular ones (by disabling the gzip option: ```npm start -- --no-gzip```).
```

Comment:
Remove the triple backticks, just use one

Reply:
I will sit with you later this noon so that we improve the message here

Reply:
Doing the compression in **Node.js** might hold the event loop which it not desirable for performance reasons. That's why we employ gzip compression at build time. We use [compression-webpack-plugin](https://github.com/webpack-contrib/compression-webpack-plugin) to compress the assets when the project is built and the production server will attempt to serve the compressed files based on the requests [Accept-Encoding](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Accept-Encoding) header.

Nevertheless, if you want to do compression in an upstream server, like **nginx**, you may disable serving the gzip files by starting the server with:

```sh
npm start -- --no-gzip
```
```